### DISCUSSION TITLE
Adding raw data into the head

### DISCUSSION THREAD

**Discussion by zcs02468** [nuxt/nuxt#18245] [Questions]
that they can use to add code to the site. The content could look like this. The exact content is unknown for example:

`<link rel="stylesheet" type="text/css" href="//a.com/a.css" />
<script src="//a.com/a.js"></script>
<script>
  console.log("111111111")
</script>`

**Comment by wobsoriano** [comment]
```ts
import { defineNuxtConfig } from 'nuxt3'

export default defineNuxtConfig({
    meta: {
        script: [
            { src: '//a.com/a.js' },
            { src: '~/assets/script.js' }
        ]
    },
})
```

**Reply by zcs02468** [reply]
The point I'm trying to make is that the inserted code is unknown, and the questions are just examples

**Reply by wobsoriano** [reply]
Where are the questions?
